newIDE/app/src/MainFrame/EditorContainers/DebuggerEditorContainer.js

```diff
@@ -24,9 +24,8 @@ export class DebuggerEditorContainer extends React.Component<
   };
 
   shouldComponentUpdate(nextProps: RenderEditorContainerProps) {
-    // Prevent any update to the editor if the editor is not active,
-    // and so not visible to the user.
-    return nextProps.isActive;
+    // We render updates only if the component is active, or becoming active.
```

Comment:
or if the component is becoming inactive (which is what solves our problem here ;)). Important to call this out.

```suggestion
    // We stop updates when the component is inactive. 
    // If it's active, was active or becoming active again we let update propagate.
    // Especially important to note that when becoming inactive, a "last" update is allowed.
```